templates/index.html

```diff
@@ -90,7 +89,7 @@ <h3 class="p-heading--2">Cloud, your way</h3>
             <hr class="p-rule is-dark u-hide--medium">
             <p class="p-heading--2"><a href="https://ubuntu.com/ceph">Ceph&nbsp;&rsaquo;</a></p>
             <hr class="p-rule is-dark u-hide--medium">
-            <p class="p-heading--2"><a href="https://ubuntu.com/lxd">LXD&nbsp;&rsaquo;</a></p>
+            <p class="p-heading--2"><a href="https://canonical.com/microcloud">MicroCloud&nbsp;&rsaquo;</a></p>
```

Comment:
```suggestion
            <p class="p-heading--2"><a href="/microcloud">MicroCloud&nbsp;&rsaquo;</a></p>
```
No need for the full url on internal links